sympy/stats/tests/test_symbolic_probability.py

```diff
@@ -93,6 +94,16 @@ def test_probability_rewrite():
     assert Expectation(Z).rewrite(Probability) == Sum(z*pz, (z, 0, oo))
     assert Variance(X).rewrite(Probability) == Integral(x**2*px, (x, -oo, oo)) - Integral(x*px, (x, -oo, oo))**2
     assert Variance(Z).rewrite(Probability) == Sum(z**2*pz, (z, 0, oo)) - Sum(z*pz, (z, 0, oo))**2
+    assert Covariance(w, X).rewrite(Probability) == \
+           -w*Integral(x*Probability(Eq(X, x)), (x, -oo, oo)) + Integral(w*x*Probability(Eq(X, x)), (x, -oo, oo))
+
+    # To test rewrite as sum function
+    assert Variance(X).rewrite(Sum) == Variance(X).rewrite(Integral)
```

Comment:
I think it is probably better to test with the explicit result here. Basically, if it is not possible to rewrite to neither Sum or Integral, this test will also pass...

Reply:
I wrote the test this way because the function is defined as:

https://github.com/sympy/sympy/blob/3cb3e34f16f764da7b58af2c983382a5578cd9df/sympy/stats/symbolic_probability.py#L272

Similar reasons for the next test for `Expectation(X)`.

But I can change these to test explicit results.

Reply:
I tried to write some explicit tests, they didn't work.

Reply:
I think what @oscargus means is that you should spell out the expression on the right-hand side explicitly rather than comparing the output of two function calls (what if they're both wrong?).

Reply:
Correct! You can just do `str(left hand side)` to get what to put on the right hand side.

As @oscarbenjamin points out both can be wrong or, as in the case of `rewrite`, an unsuccessful rewrite will just return the original expression. It can still be worthwhile to test that it does though, for example to increase test coverage, but if you do, say `f.rewrite(gamma) == f.rewrite(sin)` it will for many functions be `True` since neither rewrite is possible. Hence, better here to do

```
expected == ...
assert Variance(x).rewrite(Integral) == expected
assert Variance(x).rewrite(Sum) == expected
```

Reply:
ok, I got it
I will make the changes with the other ones in the next push